### ISSUE TITLE
Cannot find file 'Exports.js'

### ISSUE THREAD

**Issue by AnEpicName** [jeffreylanters/react-unity-webgl#123]
Hi, I'm having a very annoying problem trying to import Unity from the npm package...

**Describe the bug**
I'm stuck getting this error: 
``` bash
./src/components/UnityView/UnityView.js
Cannot find file: 'Exports.js' does not match the corresponding name on disk: '.\node_modules\react-unity-webgl\distribution\exports.js'.
``` 
And no matter what I try, I'm always getting the same.

**To Reproduce**
Basically I'm following the basic quick start guide and I'm just doing this:
``` javascript 
import Unity from 'react-unity-webgl' 
```
I didn't even begin to do a more complex behaveour and it's weird because I've used this package before and have never has a problem like this and I don't know if I'm missing something or did something wrong...

I installed the package just using this as always:
``` bash
npm install react-unity-webgl
```

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro
 - Browser: Chrome
 - Version: 85.0.4183.121

**Comment by jeffreylanters** [comment]
Hi! Can you try a clean install?

**Comment by AnEpicName** [comment]
Hi @jeffreylanters ! Yes, I fixed it by deleting the node_modules folder and doing the npm install command again and that was it, so thanks!

**Comment by fezproof** [comment]
Hi @jeffreylanters , I am having this issue still with version 7.1.12 of the package. I have reinstalled a few times and confirmed it is definitely still an issue. I downgraded to 7.1.10 and this seems to fix the issue, but it would be nice to use the latest LTS release.

7.1.10
![image](https://user-images.githubusercontent.com/26830309/123201507-67dbf600-d4e5-11eb-8900-4a12792c818f.png)

7.1.12
![image](https://user-images.githubusercontent.com/26830309/123201620-9bb71b80-d4e5-11eb-987e-f6e0afc3fc86.png)

Would be happy to help fix it :) I am just not sure what exactly is going on

**Comment by jeffreylanters** [comment]
Hi! Thanks for reporting this problem. I think this is an issue caused by Git. I'll push an hotfix later today.

**Comment by fezproof** [comment]
> Hi! Thanks for reporting this problem. I think this is an issue caused by Git. I'll push an hotfix later today.

Awesome, thanks for responding so quickly ♥

**Comment by jeffreylanters** [comment]
No problem at all, sorry for the inconvenience! I've just pushed an hot-fix in version `7.1.13` over to the package registry. Could you try this and let me know if it's working? Thanks!

**Comment by fezproof** [comment]
Looks good to me! Amazing turnaround thankyou :D

![image](https://user-images.githubusercontent.com/26830309/123234970-08461080-d50e-11eb-8578-12b4cd7cdc6b.png)
